xc/xc7/tests/simple_ff/CMakeLists.txt

```diff
@@ -51,4 +51,6 @@ add_custom_target(
   DEPENDS ${OUT_FASM}
   )
 
-add_dependencies(all_xc7_tests simple_ff_full_loc_check)
+# XXX: need to fix attribute trimming when running with abc9
```

Comment:
This isn't required anymore is it?  There was a fix upstream?

Reply:
I need to double check this, but this should be a different issue w.r.t. the one that did not allow the BUFGCTRL or PLL to be LOCed. This issue seems to be related on the new code that handles FF for ABC in yosys.

Reply:
So we still need to fix yosys before we integrate it?  It still has outstanding issues that prevent it from being usable?

Reply:
It is currently usable for the regression tests within symbiflow-arch-defs, apart from this last one. I am in the process of verifying once again that the issue is still present.

Reply:
This issue is actually solved, the LOC constraint is present after synthesis. I will re-enable the test.

Reply:
Awesome!